Package.swift

```diff
@@ -30,6 +30,18 @@ let package = Package(
             name: "WorkflowCombine",
             targets: ["WorkflowCombine"]
         ),
+        .library(
+            name: "WorkflowTesting",
+            targets: ["WorkflowTesting"]
+        ),
+        .library(
+            name: "WorkflowReactiveSwiftTesting",
+            targets: ["WorkflowReactiveSwiftTesting"]
+        ),
+        .library(
+            name: "WorkflowCombineTesting",
+            targets: ["WorkflowCombineTesting"]
+        ),
     ],
     dependencies: [
         .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift.git", from: "6.3.0"),
```

Comment:
`.package(url: "https://github.com/nicklockwood/SwiftFormat", .exact("0.44.14")),`

Is there a reason we are pinning the SwiftFormat version?
Should generally use `from` and then depend on a Package.resolved file

Reply:
@dhavalshreyas Do you know what this dependency is pinned at an exact version?